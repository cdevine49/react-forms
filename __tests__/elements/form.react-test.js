import React from 'react';
import Form from '../../lib/elements/form';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

test("renders with default props", () => {
  const tree = renderer.create(
    <Form
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

test("accepts props", () => {
  const tree = renderer.create(
    <Form
      id="formId"
      className="new-class"
      noValidate
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});