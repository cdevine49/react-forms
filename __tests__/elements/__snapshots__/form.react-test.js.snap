// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`accepts props 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 400px;
  padding: 20px;
}

<form
  className="new-class c0"
  id="formId"
  noValidate={true}
  onSubmit={[Function]}
/>
`;

exports[`renders with default props 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 400px;
  padding: 20px;
}

<form
  className="c0"
  onSubmit={[Function]}
/>
`;
